### DISCUSSION TITLE
申请退款一个多月了，还没处理？？？

### DISCUSSION THREAD

**Discussion by vincentye9682** [getlantern/lantern-forum-cn#1080] [General]
邮箱: yizunquan@163.com

Payment time: 2024-11-07 11:20:28

Payment method: CMB Credit Card(7419)

游戏充值订单号: P01202411071119396916555

Full name of payee: 广州饶铭网络科技有限公司

Order No.: 2024110722001476771458238537

Merchant order No.: 2024110711194145426

![391765796-a79c6447-e78b-48c3-9b67-d7b9f7b5528f](https://github.com/user-attachments/assets/1124c200-374b-4a15-af10-ead6dc5f40ac)

**Comment by MaxMKMax** [comment]
@vincentye9682 您好. 早就已经提交上去了. 我再帮您跟进一下.

**Reply by vincentye9682** [reply]
好，谢谢
